### PR TITLE
Run tests on alphanet_v1 and testnet_v1 every 6 hours

### PR DESCRIPTION
**Pull request by Flouse**
```yaml
contract-tests:
    strategy:
      fail-fast: false
      matrix:
        network: [gw_testnet_v1, gw_alphanet_v1]
```

### CODE REVIEW

File: .github/workflows/contract-test.yml [hunk 1]
```diff
@@ -58,7 +62,7 @@ jobs:
       run: npm install
     - name: Run tests
       working-directory: contracts
-      run: npm run test:gw_testnet_v1
+      run: npm run test:${{ matrix.network }}
```
Comment (ShookLyngs) [inline .github/workflows/contract-test.yml:65]: Should we use different private keys for different networks?

Comment (Flouse) [reply .github/workflows/contract-test.yml:65]: Not necessary since they are different layer2 networks, so conflicts will not happen.

**Review by ShookLyngs — APPROVED** [review]

File: .github/workflows/contract-test.yml (hunk 1 above)
Comment (ShookLyngs) [reply .github/workflows/contract-test.yml:65]: Maybe then we should change the `GODWOKEN_TESTNET_PRIVATE_KEY` secrets' name to something else like `GODWOKEN_TEST_PRIVATE_KEY` to better suite the case? 

Do you thinks it's necessary? Neither way is urgent tho.

Comment (Flouse) [reply .github/workflows/contract-test.yml:65]: Yes, please update the names of the secrets in another PR when you have a moment.

**Pull request merged**